libraries/adaptive-expressions/src/builtinFunctions/take.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * @module adaptive-expressions
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ExpressionEvaluator } from '../expressionEvaluator';
+import { Expression } from '../expression';
+import { ReturnType } from '../returnType';
+import { ExpressionType } from '../expressionType';
+import { FunctionUtils } from '../functionUtils';
+import { Options } from '../options';
+
+/**
+ * Return items from the front of a collection.
```

Comment:
>collection [](start = 36, length = 10)

We are very inconsistent in terminology, i.e. we use list, array and collection.  I would suggest we use array everywhere.